a.core_concepts.md

```diff
@@ -71,7 +71,7 @@ kubectl run nginx --image=nginx --restart=Never --dry-run -o yaml | kubectl crea
 </p>
 </details>
 
-### Create a busybox pod (using kubectl command) that runs the command "env". Run it and see the output
+### Create a BusyBox pod (using kubectl command) that runs the command "env". Run it and see the output
```

Comment:
Name of the project is BusyBox, instead of busybox, not trying to be too pedantic but I find it easier to read, specially since the pod name and image in this case are identical.

Reply:
You're right. Can we add a comment that the image name is lowercase, though (as all repository names)? Would make life easier for people new to K8s